### PR TITLE
{compiler}[foss/2019b] pythran v0.9.4.post1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node375.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e25386ca73ee53b121c29fc0379218b4 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/2a26afdf149557ac3082d382813ebb0b for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @smoors!

File: easybuild/easyconfigs/p/pythran/pythran-0.9.4.post1-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
+
+modloadmsg = """
+To use the OpenBLAS library, run:
+pythran --config=compiler.blas=openblas <python_script.py>
```
Comment (boegel) [inline easybuild/easyconfigs/p/pythran/pythran-0.9.4.post1-foss-2019b-Python-3.7.4.eb]: @smoors Maybe it's better to define an alias for the `pythran` command for this in the module, so people can just use `pythran <python_script.py>`?

Comment (smoors) [reply easybuild/easyconfigs/p/pythran/pythran-0.9.4.post1-foss-2019b-Python-3.7.4.eb]: @boegel good idea, see #9594

**Pull request merged**